omega1s1.red

```diff
@@ -1,11 +1,10 @@
 import path
 import integer
-import univalence
 
 let S1-univ-cover (x : S1) : type =
   S1-rec x with
   | base => int
-  | loop i => UA int int succ-equiv i
+  | loop i => `(V i int int succ-equiv)
```

Comment:
I suppose a complete solution is to have the `V` tactic... not my priority now though.

Reply:
Yes, that'll be easy to add, and I'll do it soon if nobody gets around to it.